std-bits/base/src/main/java/org/enso/base/file_format/TextFormatSPI.java

```diff
@@ -0,0 +1,14 @@
+package org.enso.base.file_format;
+
+@org.openide.util.lookup.ServiceProvider(service = FileFormatSPI.class)
+public class TextFormatSPI extends FileFormatSPI {
+  @Override
+  protected String getModuleName() {
```

Comment:
An alternative to overriding these two methods would be to define two argument constructor:
```
protected FileFormatSPI(String moduleName, String typeName);
```
that would store these two values in a private fields and used them from `get_types` method.